src/Questions/math.js

```diff
@@ -0,0 +1,28 @@
+import Questions from './questions';
+
+const mathQuestions = [
+    {
+        question: 'What is 1 + 1?',
+        answer: 2,
+    },
+    {
+        question: 'What is 1 + 2?',
+        answer: 3,
+    },
+    {
+        question: 'What is 1 + 3?',
+        answer: 4,
+    },
+    {
+        question: 'What is 1 + 4?',
+        answer: 5,
+    },
+    {
+        question: 'What is 1 + 5?',
+        answer: 6,
+    },
+];
+
+const Math = new Questions(mathQuestions);
+
+export default Math;
```

Comment:
I think it's a little confusing we're using `Math` as the name here, since it's a global keyword for `Math` object already being used for things like `Math.floor`.

Reply:
Good call, updated it on my end and will push to this branch after going through your other comments.